zenoh_cpp_vendor/CMakeLists.txt

```diff
@@ -39,8 +42,11 @@ ament_vendor(zenoh_cpp_vendor
   VCS_URL https://github.com/eclipse-zenoh/zenoh-cpp
   VCS_VERSION e8eca99b4eaff0963e52aefd8405909c1552080d
   CMAKE_ARGS
-    -DZENOHCXX_ZENOHC=OFF
-)
+      -DZENOHCXX_ZENOHC=ON
+      -DZENOHCXX_ZENOHPICO=OFF
+      -DZENOHC_BUILD_WITH_SHARED_MEMORY:BOOL=${RMW_ZENOH_BUILD_WITH_SHARED_MEMORY}
+      -DZENOHC_BUILD_WITH_UNSTABLE_API=TRUE
+  )
 
```

Comment:
Why don't we build zenoh-c with SHM? zenoh-cpp does not contain a flag for `ZENOHC_BUILD_WITH_SHARED_MEMORY` nor `ZENOHC_BUILD_WITH_UNSTABLE_API`.

Reply:
right, removed